### ISSUE TITLE
Should this repo auto-close issues with unmodified templates?

### ISSUE THREAD

**Issue by dagood** [dotnet/core#4166]
It seems like there are a decent number of issues filed here where:

1. The issue title contains no useful details. For example, `dotnet` or `dotnet run`.
2. The issue description is an unmodified template from https://dot.net.
3. The user doesn't respond when asked for details.

I think it would make sense to have a bot auto-close issues matching (2), with a generic response asking for more information. If the user responds, the bot should re-open the issue. (If it didn't reopen the issue, the user may be left in limbo with nobody able to see their question.)

I'm not sure how to accomplish this and there's no concrete plan to do this, but wanted to put it out there for discussion.

/cc @scalablecory @dleeapho @leecow

**Comment by carlossanlop** [comment]
@scalablecory and I were talking about this recently.

@richlander and @terrajobst may have different opinions about automatically closing issues.

I don't mind having the issues autoclosed. If no one objects, we could ask @maryamariyan to help set up a bot task that detects issues in this repo containing the default template text in the description (with no modifications) and close them automatically with a reply stating the reason.

**Comment by scalablecory** [comment]
I'm not against it but I do feel like this is just a band-aid to hide a symptom. It would be better for our users to find whatever UX problem is causing people to file these useless issues. It's not clear yet if these are genuine asks for help, [accidental button clicks](https://github.com/dotnet/website/issues/1703), etc.

I think using Github's default issue template is part of the problem. I see a lot of issues where people change the issue title to something vague, but never change the body. This indicates to me that many people *are* wanting help, but aren't understanding how the issue process works. Perhaps they believe some Watson-like process is happening. Perhaps English is not their native language and they can't read the template.

**Comment by scalablecory** [comment]
CC @karelz

**Comment by karelz** [comment]
Agreed, we should try to identify the "why" first nd then create solution(s) based on that.

**Comment by carlossanlop** [comment]
@mairaw I think the tutorials that link to the dotnet/core repo to report issues might fall in your new area. Is this a correct assumption?

**Comment by mairaw** [comment]
Let me look into this @carlossanlop. I didn't know people were sent here for website issues. 

I have implemented the no-response bot for the docs repos and it's been working super well.
You can see an example here: https://github.com/dotnet/docs/issues/16868

**Comment by dagood** [comment]
The templates here are specified via URL, so the website has total control:

https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/create =>
`https://github.com/dotnet/core/issues/new?labels=area-tutorials&body=Problem encountered on https%3A%2F%2Fdotnet.microsoft.com%2Flearn%2Fdotnet%2Fhello-world-tutorial%2FcreateOperating System%3A windowsProvide details about the problem you are experiencing. Include your operating system version%2C exact error message%2C code sample%2C and anything else that is relevant.`

**Comment by mairaw** [comment]
Yep just checked that. I'll start triaging issues here then.

I think auto closing is a good idea after a certain period of time with no response. The bot we use is based on the needs-more-info label. Some folks will reply when they see the issue being closed and will respond, which makes the issue reopen with that bot.

**Comment by carlossanlop** [comment]
I find it interesting that the tutorial has been modified to include a message describing the probable root cause for the common problem of "dotnet not being recognized", yet we keep getting those kinds of reports.

Maybe an "IMPORTANT" header could call the attention of people a bit more (like the ones we have in MS Docs).

**Comment by mairaw** [comment]
I've opened an issue to track this specific problem @carlossanlop.

**Comment by mairaw** [comment]
I don't have permissions on this repo but I'd love to get the auto closing bot here for case #3 (The user doesn't respond when asked for details.)

**Comment by mairaw** [comment]
Issue 3 is solved. Do we still want to do this for 1 and 2?

**Comment by mairaw** [comment]
Ping @dagood @carlossanlop

**Comment by carlossanlop** [comment]
We definitely need to get issues with unmodified templates to be automatically closed (no. 2). I vote we do that one.

Maybe no. 1 can have a bad title, but a good description. Unless they did not modify the template, then it should fall into the no. 2 category.
